orion/core.go

```diff
@@ -311,7 +311,7 @@ func (d *DefaultServerImpl) signalWatcher() {
 				}
 				info.sf.DisposeService(info.ss, params)
 			}
-		} else if sig == syscall.SIGTERM {
+		} else if sig == syscall.SIGTERM || sig == syscall.SIGINT {
```

Comment:
Who will be sending SIGINT @vvarma

Reply:
I believe this will be sent on shutdown of the system (also ctrl+c) for graceful termination